.github/workflows/beam_PreCommit_Yaml_Xlang_Direct.yml

```diff
@@ -91,7 +91,7 @@ jobs:
       - name: run PreCommit Yaml Xlang Direct script
         uses: ./.github/actions/gradle-command-self-hosted-action
         with:
-          gradle-command: :sdks:python:yamlIntegrationTests
+          gradle-command: :sdks:python:yamlIntegrationTests -PbeamPythonExtra=ml_test
```

Comment:
As we add more tests, it may worth split out the workflow and create a "PreCommit YAML Xlang" exercises more tests on postsubmit schedule, to keep presubmit run on reasonable time. Could be a follow up